### PR TITLE
MXRT600 add sleep mode support

### PR DESCRIPTION
**Pull request by mmahadevan108**
This dependent on PR https://github.com/zephyrproject-rtos/zephyr/pull/34086

### CODE REVIEW

**Comment by mmahadevan108** [comment]
Any feedback?

**Comment by mmahadevan108** [comment]
@hakehuang

File: west.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ manifest:
       revision: f49bd1354616fae4093bf36e5eaee43c51a55127
       path: tools/net-tools
     - name: hal_nxp
-      revision: 27a464e4885f393dc376a53d9fc8d52078848496
+      revision: pull/88/head
```
Comment (MaureenHelm) [inline west.yml:103]: The hal PR has now been merged, so you can update this to the new SHA

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: west.yml (hunk 1 above)
Comment (mmahadevan108) [reply west.yml]: Done

**Review by ceolin — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**